### PR TITLE
Add breaking change alert for EspMilightHub

### PR DESCRIPTION
**Pull request by ccutrer**
See https://github.com/openhab/openhab-addons/pull/13801

### CODE REVIEW

**Review by wborn — CHANGES_REQUESTED** [review]
It should be moved to the right OH version when it gets merged.

**Comment by lolodomo** [comment]
@ccutrer : is there something else to change here ? Can you remove the draft status so that it can be merged ?

**Comment by ccutrer** [comment]
Rebased and put into the proper section. Ready to merge now.

File: distributions/openhab/src/main/resources/bin/update.lst [hunk 1]
```diff
@@ -127,6 +127,7 @@ ALERT;Miele@home Binding: The channel 'powerConsumption' has been renamed to 'en
 ALERT;Twitter Binding has become X. All related references to Twitter and Tweets have been replaced, please check your configuration (rules, actions, channels etc).
 ALERT;Porcupine Keyword Spotter: removed from the distribution, it was unmaintained.
 ALERT;Action Template Interpreter: removed from the distribution, functionality is covered by the Standard Interpreter.
+ALERT;MQTT EspMilightHub Binding: Colour bulbs no longer have a `level` channel. You may need to re-add your things for it to be removed. Link to the `colour` channel instead, even for `Dimmer` items.
```
Comment (lolodomo) [inline distributions/openhab/src/main/resources/bin/update.lst:130]: We try to present them in alphabetic order.
Please move up your line.
And if you do it also for Porcupine and Action Template Interpreter, that would be great.

**Review by wborn — APPROVED** [review]
Thanks!

**Pull request merged**